c2corg_ui/static/js/card.js

```diff
@@ -67,12 +65,18 @@ app.CardController = function(gettextCatalog) {
   this.doc;
 
   /**
+   * @type {string}
+   * @public
+   */
+  this.type = app.utils.getDoctype(this.doc['type']);
+
+  /**
    * @type {Object}
    * @export
    */
-  this.locale = this.doc['locales'][0];
-  for (var i = 0, n = this.doc['locales'].length; i < n; i++) {
-    var l = this.doc['locales'][i];
+  this.locale = this.doc.locales[0];
+  for (var i = 0, n = this.doc.locales.length; i < n; i++) {
+    var l = this.doc.locales[i];
```

Comment:
You're definitely right. I have just made a change with 55d2eb4

It seemed to work anyway since I previously had no warning when compiling and the cards locales were correctly fed.